prime-router/src/main/kotlin/fhirengine/translation/hl7/utils/ConstantResolver.kt

```diff
@@ -178,6 +182,10 @@ class FhirPathCustomResolver : FHIRPathEngine.IEvaluationContext, Logging {
     ): MutableList<Base> {
         check(focus != null)
         return when {
+            GlobalCustomFHIRFunctions.resolveFunction(functionName) != null -> {
```

Comment:
We don't want this global vs custom since within the library there is no concept of global. Keep the CustomFHIRFunction class as it was and add you LIVD table function until we separate it. This will also make the review smaller since you did not change any of that code. Just comment your function to say that will be moved out of the library in a next ticket.